default/stringtables/en.txt

```diff
@@ -4705,6 +4705,29 @@ Important sections include [[encyclopedia CATEGORY_GAME_CONCEPTS]] and the recen
 
 '''
 
+SITREP_DISMISS_HIDE_TITLE
+Dismissing and Hiding SitReps
+
+SITREP_DISMISS_HIDE_SHORT_DESC
+Dismissing a specific SitRep and hiding a SitRep template.
```

Comment:
I dislike the use of the word template, I don't think it's appropriate, but I'm not sure what would be better. Sitrep type?